Workbooks/WorkbooksMetadata.json

```diff
@@ -1088,5 +1088,18 @@
     "templateRelativePath": "IntsightsIOCWorkbook.json", 
     "subtitle": "",
     "provider": "Intsights Cyber Intelligence"
-  }
+  },
+  {
+ "workbookKey": "ForcepointCloudSecurityGatewayWorkbook",
+ "logoFileName": "Forcepoint_new_logo.svg",
+ "description": "Use this report to understand query runs across your workspace.",
+ "dataTypesDependencies": ["CommonSecurityLog"],
+ "dataConnectorsDependencies": [],
```

Comment:
Please Update "dataConnectorsDependencies" as  data connector id value in the data connector(s) on which this workbook depends on. like [ "ForcepointCloudSecurityGateway" ]